### PR TITLE
readme: add restore/backup walkthrough to readme

### PR DESCRIPTION
**Pull request by fanminshi**
[skip ci]

### CODE REVIEW

**Comment by fanminshi** [comment]
cc/ @hongchaodeng @hasbro17

File: README.md [hunk 1]
```diff
@@ -267,6 +269,14 @@ $ kubectl get pod example-etcd-cluster-0000 -o yaml | grep "image:" | uniq
 
 Check the other two pods and you should see the same result.
 
+## Backup an etcd cluster
+
+follow [etcd backup operator walkthrough](./doc/user/walkthrough/backup-operator.md) to backup an etcd cluster.
```
Comment (hasbro17) [inline README.md]: Follow the ...

File: README.md [hunk 2]
```diff
@@ -267,6 +269,14 @@ $ kubectl get pod example-etcd-cluster-0000 -o yaml | grep "image:" | uniq
...
+
+## Restore an etcd cluster from backup
+
+follow [etcd restore operator walkthrough](./doc/user/walkthrough/restore-operator.md) to restore an etcd cluster on Kubernetes from backup.
```
Comment (hasbro17) [inline README.md]: Follow the ...

**Comment by hasbro17** [comment]
LGTM

**Comment by fanminshi** [comment]
all fixed.

**Pull request merged**